README.md

```diff
@@ -1,214 +1,214 @@
-| Object Detection | Detect objects in an image using a bounding box or polygons |
-| Semantic Segmentation | Detect for each pixel the object category it belongs to | 
-| Pose Estimation | Mark positions of a person’s joints |
+|----|----|
+| **Images** | | 
+| [Classification](https://labelstud.io/templates/image_classification.html) | Categorize images |
+| Object Detection | Identify objects in an image using a bounding box or polygons |
+| Semantic Segmentation | Detect the object category for each pixel in an image | 
+| Pose Estimation | Mark the positions of a person’s joints |
 | **Text** | | 
-| [Classification](https://labelstud.io/templates/sentiment_analysis.html) | Put texts into categories |
+| [Classification](https://labelstud.io/templates/sentiment_analysis.html) | Categorize the content or sentiment of text |
 | Summarization | Create a summary that represents the most relevant information within the original content |
-| HTML Tagging | Annotate things like resumes, research, legal papers and excel sheet converted to HTML | 
+| HTML Tagging | Annotate things like webpages, as well as resumes, research, legal papers, and spreadsheets converted to HTML | 
+| Named Entity Recognition | Annotate specific portions of text |
 | **Audio** | |
-| [Classification](https://labelstud.io/templates/audio_classification.html) | Put audios into categories |
-| Speaker Diarisation | partitioning an input audio stream into homogeneous segments according to the speaker identity | 
-| Emotion Recognition | Tag and identifying emotion from the audio |
-| Transcription | Write down verbal communication in text |
+| [Classification](https://labelstud.io/templates/audio_classification.html) | Categorize audio content |
+| Speaker Diarisation | Partition an audio stream into homogeneous segments according to the speaker identity | 
+| Emotion Recognition | Tag and identify the emotion in the audio |
+| Transcription | Convert the speech in the audio to text |
 | **Video** | |
 | [Classification](https://labelstud.io/templates/video_classification.html) | Put videos into categories | 
 | **Comparison** | |
 | Pairwise | Comparing entities in pairs to judge which of each entity is preferred | 
 | Ranking | Sort items in the list according to some property |
 | **Time Series** | |
-| Classification |  |
-| Segmentation |  |
+| Classification | Categorize the types of events occurring over time |
+| Segmentation | Separate the portions of a time series event in a useful way |
 
-## Machine Learning Integration
 
-You can easily connect your favorite machine learning framework with Label Studio Machine Learning SDK. It's done in the simple 2 steps:
-1. Start your own ML backend server ([check here for detailed instructions](label_studio/ml/README.md)),
-2. Connect Label Studio to the running ML backend on [/model](http://localhost:8080/model.html) page
+## Integrate with machine learning models
```

Comment:
There will be two very similar topics about machine learning integration. Check it on rendered version of this readme file - https://github.com/heartexlabs/label-studio/tree/docs/readme-updates

Reply:
Yeah, @niklub had the same comment with how it appears in the list too. Check https://github.com/heartexlabs/label-studio/pull/579#discussion_r569821167

Reply:
Oh, sorry.